sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -69,7 +69,15 @@ public LogsQueryClient(Uri endpoint, TokenCredential credential, LogsQueryClient
 
             Endpoint = endpoint;
             options ??= new LogsQueryClientOptions();
-            var scope = $"{endpoint.AbsoluteUri}/.default";
+            string scope;
```

Comment:
Why is the logic different from ingestion - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs#L38